### PR TITLE
Fixes magboots eating shoes and nutriments dealing toxin damage

### LINKED ISSUE #4378: Research folders, inappropriate description
Research folders are inaccurately described as "gold and blue", after being resprited to be purple and white.

image

image

### LINKED ISSUE #4379: Hardsuits / Rigs / Gloves (Covering Items) eat shoes / rings
After using a hardsuit or rig your shoes will vanish into bluespace. They leave a ghost icon (and ghost code) of shoes-past on the UI and game logic however by all purposes they're gone.

Equipping another pair of shoes over the ghost shoes does work aswell.

### PR DESCRIPTION
**Pull request by Alberyk**
Fixes #4379
Fixes #4378 
Fixes ninja's shoes not being magboots

### CODE REVIEW

File: code/modules/mob/inventory.dm [hunk 1]
```diff
@@ -38,7 +38,6 @@
 //This is an UNSAFE proc. It merely handles the actual job of equipping. All the checks on whether you can or can't eqip need to be done before! Use mob_can_equip() for that task.
 //In most cases you will want to use equip_to_slot_if_possible()
 /mob/proc/equip_to_slot(obj/item/W as obj, slot)
-	W.on_slotmove(src)
```
Comment (Arrow768) [inline code/modules/mob/inventory.dm]: That will screw with the deadman trigger.
Please continue to track it down the call stack and see if the issue with the shoes can be fixed some other way.

Comment (Alberyk) [reply code/modules/mob/inventory.dm]: I did tests without this proc here, and the deadman switch did work just fine, even when you equipped the signaler it into your pocket.

Comment (Alberyk) [reply code/modules/mob/inventory.dm]: Done.

**Comment by Arrow768** [comment]
It seems you have fixed something that has been fixed with: #4389

**Comment by Alberyk** [comment]
Fixed the conflict.

**Pull request closed without merging**